scripts/translations.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
+const path = require('path');
+
+const shell = require('shelljs');
+
+const main = async () => {
+  const {stdout} = shell.exec('git diff --name-only --cached', {silent: true});
```

Comment:
🤔 What about if I do a commit not in the root of the repository?; Like if I'm working on a plugin and I'm on `/packages/strapi-plugin-some/` ?? 🤔